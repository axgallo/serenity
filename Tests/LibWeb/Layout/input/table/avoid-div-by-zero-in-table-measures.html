<style>
	td {
		border: 1px solid black;
	}
</style>
<table>
	<tr>
		<td colspan="2">A</td>
	</tr>
</table>