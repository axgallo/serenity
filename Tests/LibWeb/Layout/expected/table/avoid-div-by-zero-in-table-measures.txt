Viewport <#document> at (0,0) content-size 800x600 children: not-inline
  BlockContainer <html> at (0,0) content-size 800x600 [BFC] children: not-inline
    BlockContainer <body> at (8,8) content-size 784x25.46875 children: not-inline
      TableWrapper <(anonymous)> at (8,8) content-size 24.265625x25.46875 [BFC] children: not-inline
        Box <table> at (8,8) content-size 24.265625x25.46875 table-box [TFC] children: not-inline
          BlockContainer <(anonymous)> (not painted) children: inline
            TextNode <#text>
          Box <tbody> at (8,8) content-size 18.265625x21.46875 table-row-group children: not-inline
            Box <tr> at (10,10) content-size 18.265625x21.46875 table-row children: not-inline
              BlockContainer <(anonymous)> (not painted) children: inline
                TextNode <#text>
              BlockContainer <td> at (12,12) content-size 16.265625x17.46875 table-cell [BFC] children: inline
                line 0 width: 14.265625, height: 17.46875, bottom: 17.46875, baseline: 13.53125
                  frag 0 from TextNode start: 0, length: 1, rect: [12,12 14.265625x17.46875]
                    "A"
                TextNode <#text>
              BlockContainer <(anonymous)> (not painted) children: inline
                TextNode <#text>
            BlockContainer <(anonymous)> (not painted) children: inline
              TextNode <#text>

ViewportPaintable (Viewport<#document>) [0,0 800x600]
  PaintableWithLines (BlockContainer<HTML>) [0,0 800x600]
    PaintableWithLines (BlockContainer<BODY>) [8,8 784x25.46875]
      PaintableWithLines (TableWrapper(anonymous)) [8,8 24.265625x25.46875]
        PaintableBox (Box<TABLE>) [8,8 24.265625x25.46875]
          PaintableBox (Box<TBODY>) [8,8 18.265625x21.46875] overflow: [8,8 22.265625x23.46875]
            PaintableBox (Box<TR>) [10,10 18.265625x21.46875] overflow: [10,10 20.265625x21.46875]
              PaintableWithLines (BlockContainer<TD>) [10,10 20.265625x21.46875]
                TextPaintable (TextNode<#text>)
